financial-templates-lib/clients/ExpiringMultiPartyClient.js

```diff
@@ -4,6 +4,8 @@
 const { LiquidationStatesEnum } = require("@umaprotocol/common");
 const Promise = require("bluebird");
 
+const Token = artifacts.require("ExpandedERC20");
```

Comment:
Similarly to @nicholaspai point, there should be no `artifacts.require` in these clients & all ABIs should be injected as params.

Reply:
Just remove this entire section its redundant. Pass in `Token` from `liquidator/index.js`